pages/myProfile/index.js

```diff
@@ -14,46 +16,110 @@ const myProfile = ({loginName, userArray}) => {
     },
   };
   // parses loginName String to Object to be able to compare them in the checkLoggedInUser Method
-  const loginNameObject = {name: `${loginName}`};
-  const checkLoggedInUser = userArray.filter(user => {
-    return user.name == loginNameObject.name;
+  const [ownProfileUser, setOwnProfileUser] = useState();
+  useEffect(() => {
+    //first
+    const loginNameObject = {name: `${loginName}`};
```

Comment:
You don't need to write the value as template string:

```suggestion
    const loginNameObject = {name: loginName};
```

Reply:
Fixed and pushed new version.